TehanovAnton/hometask-4/my_hash.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+# this class is my realisation of Hash
+class MyHash
+  include Enumerable
+
+  LOAD_FACTOR = 0.7
+  DEFAULT_SIZE = 10
+
+  def initialize
+    initialize_hash!
+  end
+
+  def [](key)
+    pair = pair(key)
+
+    pair&.last
+  end
+
+  def []=(key, value)
+    resize! if resize?
+
+    slot = slot(key)
+    pair = pair(key, slot: slot)
```

Comment:
Do you think it is a good idea to name local variables with same names as private methods?